jetty-runtime/jetty9-base/pom.xml

```diff
@@ -122,7 +135,7 @@
           <arguments>
             <argument>-jar</argument>
             <argument>../jetty-distribution-${jetty.version}/start.jar</argument>
-            <argument>--add-to-startd=gae,http,jsp,jstl</argument>
+            <argument>--add-to-startd=http,deploy,jsp,jstl,http-forwarded,resources,gae</argument>
```

Comment:
The `gae.mod` further down moves the `deploy` and `resources` modules to `[optional]`, meaning that they will not be automatically pulled in.

Adding them to this command line makes sense if they are needed for normal operation.